src/test/java/org/example/schoology/pages/courses/Courses.java

```diff
@@ -18,4 +26,17 @@ public String getCourseTitle(final String courseName) {
         String xPath = String.format("//span[text()='%s']", courseName);
         return action.getText(By.xpath(xPath));
     }
+
+
+    public DeletePopup clickDeleteCourse(String courseName) {
+        WebElement courseActionsButton = driver.findElement(By.xpath(String.format(COURSE_ACTIONS_BUTTON, courseName)));
+
+        // Scroll
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("arguments[0].scrollIntoView();", courseActionsButton);
```

Comment:
use the method created in common actions

Reply:
Done

Reply:
done